web/src/features/panels/zone/FeedbackCard.tsx

```diff
@@ -0,0 +1,259 @@
+  };
+
+  const handleSave = () => {
+    setState('3');
+    fetch(`https://hooks.zapier.com/hooks/catch/14671709/3l9daod/`, {
+      method: 'POST',
+      body: JSON.stringify({
+        score: feedbackScore,
+        feedback: inputText,
+        reference: estimationMethod,
+      }),
+    });
+  };
+
+  if (state === '3') {
+    return null;
+  }
+
+  return (
+    <div className="flex flex-col">
+      <div data-test-id="feedback-question" className="text-sm">
+        {question}
+      </div>
+      <ActionPills setState={setState} setFeedbackScore={setfeedbackScore} />
+      {state === '2' && (
+        <div>
+          <div className="my-3 h-[1px] w-full bg-neutral-200 dark:bg-gray-700" />
+          <div>
+            <InputField inputText={inputText} handleInputChange={handleInputChange} />
+            <SubmitButton handleSave={handleSave} />
+          </div>
+        </div>
+      )}
+    </div>
+  );
+}
+
+function ActionPills({
+  setState,
+  setFeedbackScore,
+}: {
+  setState: Dispatch<SetStateAction<string>>;
+  setFeedbackScore: Dispatch<SetStateAction<string>>;
+}) {
+  const agreeText = getQuestionTranslation('agree');
+  const [pillContent] = useState([1, 2, 3, 4, 5]);
+  const disagreeText = getQuestionTranslation('disagree');
+  const [CurrentPillNumber, setPillNumber] = useState('');
+
+  const handlePillClick = (identifier: string) => {
+    setFeedbackScore(identifier);
+    setPillNumber(identifier);
+    setState('2');
+  };
+
+  return (
+    <div className="flex w-full max-w-sm flex-col pt-2">
+      <PillContent
+        pillContent={pillContent}
+        handlePillClick={handlePillClick}
+        CurrentPillNumber={CurrentPillNumber}
+      />
+      <div className="flex flex-row items-center justify-between pt-1">
+        <div
+          data-test-id="disagree-text"
+          className="text-xs font-medium text-neutral-400"
+        >
+          {disagreeText}
+        </div>
+        <div data-test-id="agree-text" className="text-xs font-medium text-neutral-400">
+          {agreeText}
+        </div>
+      </div>
+    </div>
+  );
+}
+
+function PillContent({
+  pillContent,
+  handlePillClick,
+  CurrentPillNumber,
+}: {
+  pillContent: number[];
+  handlePillClick: (identifier: string) => void;
+  CurrentPillNumber: string;
+}) {
+  return (
+    <ToggleGroupPrimitive.Root
+      className={'flex-start mb-2 flex flex-row items-center justify-between'}
+      type="multiple"
+      aria-label="Font settings"
+    >
+      {pillContent.map((content) => (
+        <ToggleGroupPrimitive.Item
+          data-test-id={`feedback-pill-${content}`}
+          key={content}
+          value={String(content)}
+          aria-label={String(content)}
+          onClick={() => handlePillClick(String(content))}
+          className={`
+          mr-2 inline-flex h-9 w-[60px] select-none items-center justify-center rounded-full border border-neutral-200 text-black  dark:border-gray-700 dark:text-white
```

Comment:
As the screen becomes larger these move quite far apart from each other, it could be nicer to fix the margin and allow the pill to grow to fill the new space. 
<img width="412" alt="image" src="https://github.com/electricitymaps/electricitymaps-contrib/assets/18622768/399b4728-f72c-4613-95e7-76cb1a5ce441">

Reply:
I would also try and center the buttons, right now they are a little to the left (just a nitpick though).

Reply:
@tonypls I've changed it so the pill since grows instead. @VIKTORVAV99 I also removed some padding so it should be centered now